src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -162,7 +162,9 @@ object TezosTypes {
   }
 
   /** root of the operation hiearchy */
-  sealed trait Operation extends Product with Serializable
+  sealed trait Operation extends Product with Serializable {
+    val internal: Boolean = false
```

Comment:
I'm almost sure this isn't needed?

Reply:
it's needed, I use it in the Operation -> OperationRow transformations

Reply:
If I look at the conversion from `block` where you actually convert the `internalRows` in
[DatabaseConversions](https://github.com/Cryptonomic/Conseil/pull/479/files#diff-36344a07cac3951e698e283f1b1bf238R462), you correctly take care to 
- convert each internal to a regular operation
- convert the result to a db operation row
- copy the result to _set internal to true_

Thus you can actually set internal to the row representation only, with no need to propagate the value in the domain model of Operations and InternalOperations

Reply:
you're right, I did not get it at first, sure, changed